Sources/SwiftDocC/SwiftDocC.docc/Resources/LinkableEntities.json

```diff
@@ -133,37 +144,45 @@
                         }
                     },
                     "kind": {
-                        "type": "string"
+                        "type": "string",
+                        "nullable": true
                     },
                     "language": {
-                        "type": "string"
+                        "$ref": "#/components/schemas/SourceLanguage",
+                        "nullable": true
                     },
                     "path": {
-                        "type": "string"
+                        "type": "string",
+                        "nullable": true
                     },
                     "title": {
-                        "type": "string"
+                        "type": "string",
+                        "nullable": true
                     },
                     "abstract": {
                         "type": "array",
                         "items": {
                             "$ref": "#/components/schemas/RenderInlineContent"
-                        }
+                        },
+                        "nullable": true
                     },
                     "usr": {
-                        "type": "string"
+                        "type": "string",
+                        "nullable": true
```

Comment:
In practice I wouldn't expect that to happen with the languages Swift-DocC currently supports, but it's technically possible so the spec needs to reflect that.

Reply:
I think the data is modeled this way because the USR is optional on the linkable entity—because non-symbols don't have USRs—and all the variant properties are optional of what the non-variant property is.